### PR TITLE
Table

### PR DESCRIPTION
**Pull request by serdec**
closes https://github.com/ClickHouse/control-plane/issues/3620

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <link
-  href="https://fonts.googleapis.com/css?family=Inter:400;700;900&display=swap"
+  href="https://fonts.googleapis.com/css?family=Inter:400;500;700;900&display=swap"
```
Comment (fneves) [inline .storybook/preview-head.html:2]: we should raise this with the design team. There are too many fonts being downloaded, they should constrain themselves to like 2 (max 3) fonts

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  <StyledHeader {...delegated}>
+    <HeaderContentWrapper>
+      {leftIcon && <Icon name={leftIcon} width="12px" height="12px" />}
+      {children}
+      {rightIcon && <Icon name={rightIcon} width="12px" height="12px" />}
+    </HeaderContentWrapper>
+  </StyledHeader>
+);
+
+const TableRow = styled.tr`
```
Comment (fneves) [inline src/components/Table/Table.tsx:49]: we should add a name to the generated component so that on story book docs it does not show up as `styled.tr`

**Review by fneves — APPROVED** [review]

File: .storybook/preview-head.html (hunk 1 above)
Comment (serdec) [reply .storybook/preview-head.html:2]: agreed

File: src/components/Table/Table.tsx (hunk 2 above)
Comment (serdec) [reply src/components/Table/Table.tsx:49]: done

**Pull request merged**